studio-docs/source/dev-graphs.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Development graphs
+sidebar_title: Local development (dev graphs)
+description: Use Studio features in your development environment
+---
+
+Development graphs (or **dev graphs**) are a special type of graph in Apollo Studio. Unlike a _deployed_ graph (which represents your data graph in deployed environments like staging and production), a dev graph represents your data graph in your own development environment. It's only visible to you, and because it's separate from your deployed graph, you can quickly and safely iterate on its schema without affecting other clients.
+
+Dev graphs provide the subset of Studio features that are most useful for development, specifically:
```

Comment:
We have talked about building out more of Studio's features into dev graphs, we just didn't start there because it's not as obvious how people would want to use our other features for local development. I wonder if we might want to gesture to our intention here though? I don't want us to close the door to dev graphs having more of Studio's features in the future.